cadasta/gui/tools/cadasta_dialog.py

```diff
@@ -34,23 +34,22 @@
 
 
 class CadastaDialog(QtGui.QDialog):
-    """ Parent class for cadasta dialog."""
+    """Parent class for cadasta dialog."""
 
     def __init__(self, parent=None):
-        """ Constructor."""
+        """Constructor."""
 
         super(CadastaDialog, self).__init__(parent)
         self.setupUi(self)
         self.message_bar = None
-        self.init_style()
+        self._init_style()
 
-    def init_style(self):
-        """ Initiate custom styles for dialog. """
-
-        self.setStyleSheet('background-color:white')
+    def _init_style(self):
```

Comment:
Hi. Looking at this commit I am just realizing that you are taking the wireframes literally perhaps. I think it is *much* better to use the defaults for styles in the dialog as it will 

a) be consistent in look and feel with the rest of QGIS
b) be compliant with HIG (Human Interface Guidelines) on each platform
c) save you from wasting a lot of time trying to get the LAF (look and feel) tweaked to look just right

So generally I would say: Don't mess with dialog styles. In terms of making things look and feel as part of the Cadasta Corporate Identity, this should be taken on board when producing  any text output (when using webkit and rich text widgets to report on status etc.